index.md

```diff
@@ -20,21 +20,6 @@ Learners are expected to have the following knowledge:
 
 ::: instructor
 
-## Looking for Beta Testers!
```

Comment:
I would suggest to instead replace this with:

## We can help you out with teaching this lesson!
Do you want to teach this lesson? Feel free to reach out to us with any questions that you have. Just [open a new issue](https://github.com/carpentries-incubator/deep-learning-intro/issues/new/choose). We also value any feedback on the lesson!

Reply:
That would seem a bit redundant with the ["Teaching this lesson?" section in the README](https://github.com/carpentries-incubator/deep-learning-intro/blob/fddec665ef5ae5885ad411246f2eaa93b972ea0e/README.md#L41). Maybe a link there would be better to keep the information in sync?

Reply:
@svenvanderburg How about [this](https://github.com/carpentries-incubator/deep-learning-intro/blob/cd26b43ed0728a4dc83d31dda2e2634f32e7e017/index.md#we-can-help-you-out-with-teaching-this-lesson)?

Reply:
Good point! I like your solution :)